Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -112,6 +120,39 @@ int LayoutAnimationsManager::findPrecedingViewTagForTransition(int tag) {
   return -1;
 }
 
+#ifdef DEBUG
+std::string LayoutAnimationsManager::getScreenSharedTagPairString(
+    const int screenTag,
+    const std::string &sharedTag) {
+  return std::to_string(screenTag) + ":" + sharedTag;
+}
+
+bool LayoutAnimationsManager::hasDuplicateSharedTag(
+    const int viewTag,
+    const int screenTag) {
+  if (!viewTagToSharedTag_.count(viewTag)) {
+    return false;
+  }
+  auto sharedTag = viewTagToSharedTag_[viewTag];
+  auto pair = getScreenSharedTagPairString(screenTag, sharedTag);
+  bool hasDuplicate = screenSharedTagSet_.count(pair);
+  if (hasDuplicate) {
```

Comment:
`count` will return the number of occurrences thus it iterates over all items. The best approach would be to use `contains` method but it's available since C++20 (we use C++17). Let's do it this way:
```suggestion
  if (screenSharedTagSet_.find(pair) != screenSharedTagSet_.end()) {
```